src/App.tsx

```diff
@@ -1,24 +1,91 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import {
+  FC,
+  useContext,
+  useEffect,
+  useMemo,
+} from 'react';
+
 import { UserWarning } from './UserWarning';
+import { TodoList } from './components/TodoList';
+import { getTodos } from './api/todos';
+
+import { Status, ErrorMessage } from './types';
+import { TodoForm } from './components/TodoForm';
+import { TodoFooter } from './components/TodoFooter';
+import { TodoError } from './components/TodoError';
+import { TodosContext } from './components/TodosContext/TodosContext';
+
+export const App: FC = () => {
+  const {
+    todos,
+    setTodos,
+    setError,
+    filterStatus,
+    USER_ID,
+  } = useContext(TodosContext);
+
+  useEffect(() => {
```

Comment:
move `useEffect` to end of the code
Would be better to use `async`  function to getTodos